CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
```

Comment:
This **is** a _test_ of [Markdown](https://en.wikipedia.org/wiki/Markdown) syntax!